python/ray/serve/controller.py

```diff
@@ -62,8 +63,17 @@ def set_shadow(self, backend: str, proportion: float):
             self.shadow_dict[backend] = proportion
 
 
-BackendInfo = namedtuple("BackendInfo",
-                         ["worker_class", "backend_config", "replica_config"])
+class BackendInfo(BaseModel):
+    # TODO(architkulkarni): Add type hint for worker_class after upgrading
+    # cloudpickle and adding types to RayServeWrappedWorker
+    worker_class: Any
```

Comment:
Is there a `Class` type?

Reply:
I'm not familiar with this, what are we looking to get out of a `Class` type?  As far as I know, in pydantic there's just `BaseModel` and `dataclass`.

Reply:
I think @edoakes  meant instead of Any can we have more specific. I think `Any` should be fine now; when cloudpickle upgrade, we can have a more specific type.